tests/integration/components/sequential-render-test.js

```diff
@@ -115,6 +118,45 @@ module('Integration | Component | sequential-render | ComponentTest', async func
       assert.verifySteps(['first', 'third', 'second']);
   });
 
+  test('Check order of execution when an getData parameter which is a promise is present', async function(assert) {
+    await render(hbs `
+      <SequentialRender
+        @renderPriority={{0}}
+        @taskName="Task1"
+        @asyncRender={{false}}
+        @renderCallback={{this.afterTask1}}
+        as |seq|
+      >
+        <seq.RenderContent>
+          <h1>Render Second</h1>
+        </seq.RenderContent>
+      </SequentialRender>
+      <SequentialRender
+        @renderPriority={{1}}
+        @taskName="task2"
+        @getData={{this.getData}}
+        @renderCallback={{this.afterTask2}}
+        as |seq1|
+      >
+      <seq1.RenderContent>
+        <h1>Render Second</h1>
+      </seq1.RenderContent>
+      </SequentialRender>
+      <SequentialRender
+        @renderPriority={{1}}
+        @taskName="task3"
+        @asyncRender={{false}}
```

Comment:
Lets remove @asyncRender here

Reply:
Here we are checking for synchronous render of component, in this case we should explicitly pass as false right ?, since we are checking for OR condition (`asyncRender || get(this, 'getData')`)